src/common/ui-components/Breadcrumb/Pill.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { IconClose, IconLocation } from 'hds-react';
+import styles from './Pill.module.scss';
+
+export interface PillProps {
+  readonly category: string;
+  readonly name: string;
+  onClose(): void;
+}
+
+const Pill: React.FC<PillProps> = ({ category, name, onClose }) => {
+  return (
+    <div className={styles.pill}>
+      <div>
+        <IconLocation className={styles.smallIcon} />
```

Comment:
Should be replace with `<CategoryIcon>` which is added in the `ListView` PR.

Reply:
That PR is merged, go ahead

Reply:
Also, your PRs are awesome

Reply:
Done. And thanks 👍